src/services/codefixes/inferFromUsage.ts

```diff
@@ -161,7 +161,7 @@ namespace ts.codefix {
                     declaration = containingFunction;
                     break;
                 }
-                // falls through
+            // falls through

```

Comment:
Can you revert this?

```suggestion
                // falls through
```

Reply:
Done, thanks for the rev!